include/deal.II/non_matching/mapping_info.h

```diff
@@ -378,13 +378,26 @@ namespace NonMatching
       const std::vector<std::vector<Quadrature<dim - 1>>> &quadrature_vector,
       const unsigned int n_unfiltered_cells = numbers::invalid_unsigned_int);
 
+    /**
+     * Compute the mapping information incoming vector of faces and
+     * corresponding vector of quadratures.
+     */
+    template <typename CellIteratorType>
+    void
+    reinit_faces(const std::vector<std::pair<CellIteratorType, unsigned int>>
+                   &face_iterator_range_interior,
+                 const std::vector<Quadrature<dim - 1>> &quadrature_vector);
+
     /**
      * Return if this MappingInfo object is reinitialized for faces (by
      * reinit_faces()) or not.
      */
     bool
     is_face_state() const;
 
+    unsigned int
+    get_face_number(const unsigned int offset, const bool is_interior) const;
```

Comment:
What does this function do?

Reply:
I have added documentation in dde60c23d8fcdc5fc7c87a13b2fb600f5522dc30